### ISSUE TITLE
🕸️ Avoir une erreur si le repo existe déjà

### ISSUE THREAD

**Issue by mat-m** [daktary-team/scribouilli#169]
J'ai tenté de créer un site via https://atelier.scribouilli.org
J'ai donné un nom à mon site. Celui-ci a été utilisé directement pour nomme le repo dans mon compte GitLab.
J'avais déjà un repo avec ce nom.

Boucle infinie sur la préparation de l'atelier (ça fait des grosses minutes :) )au lieu d'un message d'erreur. Et préciser que le nom est utilisé tel quel (je n'ai pas essayé avec des espaces, cependant).

**Comment by DavidBruant** [comment]
Oui, effectivement, créer un repo qui existe déjà ça génère une erreur et on ne produit pas de message d'erreur pour le moment

On y viendra à un moment

Merci pour le retour de bug !
